packages/core/src/components/toast/toast.scss

```diff
@@ -82,6 +82,7 @@
     flex-direction: column;
     flex: 1;
     max-width: 250px;
+    word-break: break-all;
```

Comment:
May I propose testing `word-break: break-word` as it might keep text more readable compared to current one.
Please do explore differences between break-word and break-all and so tests, it may be that break-word might be slightly better option here.